test/classes/delete-free/bad-retain-error.2.good

```diff
@@ -0,0 +1,2 @@
+bad-retain-error.chpl:13: In function 'main':
+bad-retain-error.chpl:18: error: Cannot assign to borrowed A? from unmanaged B
```

Comment:
If compopts hasn't changed, how did this case work without this file?  Was it using the `bad-retain-error.good` file?  Could/should it continue to do so?  Or should that file be removed?

Reply:
This test case relied on the fact that `owned` and `shared` were symmetric and had a retain method. Changing to adopt for the `owned` case changes the error message

Reply:
Ah, I'd missed that `bad-retain-error.1.good` was renamed (maybe in part because this summary view doesn't seem to show that?)  Thanks!

(just in case you don't know, if you don't like relying on the `.n.good` format and remembering whether it's 0-based or 1-based, you can name the .good files in the .compopts file and give them more descriptive names, like:

```
-stestShared=true   # bad-retain-error-shared.good
-stestShared=false  # bad-retain-error-owned.good
```

I'm not saying you need to do this, though...  However, I find it a much more preferable style.